### DISCUSSION TITLE
Export Zettlr files to EPUB?

### DISCUSSION THREAD

**Discussion by eduardanto** [Zettlr/Zettlr#3450] [Q&A]
Can I export my files made with Zettlr export to EPUB?

**Comment by SkampySoft** [comment]
Well, I would have said:
> Infact yes.
> with a little workaround I found:
> 
> Zettler Issue https://github.com/Zettlr/Zettlr/issues/228

The thing is. In the latest version 2.2.5 there seems to be no settings for pandoc.

Even after disabling the internal pandoc, there is no obvious way to modify the pandoc settings. https://docs.zettlr.com/en/reference/settings/#export

Am I missing something here?

**Reply by nathanlesage** [reply]
Could be that I dumped the ebook exporting format in the meantime …? If there is none it's a matter of two minutes to add it again.

In the meantime, as a workaround, you can "hijack" a setting that you don't use (e.g., RTF) and exchange the `writer` field to the corresponding eBook setting. Then, whenever you export to "rtf" you're actually exporting to ebook. You'd have to rename the file afterwards but at least it should work!
